{"level":"info","msg":"Wrapper started","time":"2021-04-28T01:23:03.816Z","wrapper":276}
{"level":"info","msg":"finding gitaly","pid_file":"/var/opt/gitlab/gitaly/gitaly.pid","time":"2021-04-28T01:23:03.816Z","wrapper":276}
{"level":"info","msg":"spawning a process","time":"2021-04-28T01:23:03.817Z","wrapper":276}
{"gitaly":281,"level":"info","msg":"monitoring gitaly","time":"2021-04-28T01:23:03.819Z","wrapper":276}
time="2021-04-28T01:23:05Z" level=info msg="Starting GitalyversionGitaly, version 13.10.3"
time="2021-04-28T01:23:05Z" level=warning msg="git path not configured. Using default path resolution" resolvedPath=/opt/gitlab/embedded/bin/git
time="2021-04-28T01:23:05Z" level=info msg="clearing disk cache object folder" path=/var/opt/gitlab/git-data/repositories
time="2021-04-28T01:23:05Z" level=info msg="moving disk cache object folder to /var/opt/gitlab/git-data/repositories/+gitaly/tmp/diskcache336507717" path=/var/opt/gitlab/git-data/repositories
time="2021-04-28T01:23:05Z" level=info msg="disk cache object folder doesn't exist, no need to remove" path=/var/opt/gitlab/git-data/repositories
time="2021-04-28T01:23:05Z" level=info msg="Starting file walker for /var/opt/gitlab/git-data/repositories/+gitaly/cache" path=/var/opt/gitlab/git-data/repositories/+gitaly/cache
time="2021-04-28T01:23:05Z" level=info msg="Starting file walker for /var/opt/gitlab/git-data/repositories/+gitaly/state" path=/var/opt/gitlab/git-data/repositories/+gitaly/state
time="2021-04-28T01:23:05Z" level=info msg="cleared all cache object files in /var/opt/gitlab/git-data/repositories/+gitaly/tmp/diskcache336507717 after 42.608Âµs" path=/var/opt/gitlab/git-data/repositories
{"level":"info","msg":"finished tempdir cleaner walk","storage":"default","time":"2021-04-28T01:23:10.544Z","time_ms":0}
{"level":"warning","msg":"spawned","supervisor.args":["bundle","exec","bin/ruby-cd","/var/opt/gitlab/gitaly","/opt/gitlab/embedded/service/gitaly-ruby/bin/gitaly-ruby","281","/var/opt/gitlab/gitaly/internal_sockets/ruby.0"],"supervisor.name":"gitaly-ruby.0","supervisor.pid":317,"time":"2021-04-28T01:23:10.567Z"}
{"level":"warning","msg":"spawned","supervisor.args":["bundle","exec","bin/ruby-cd","/var/opt/gitlab/gitaly","/opt/gitlab/embedded/service/gitaly-ruby/bin/gitaly-ruby","281","/var/opt/gitlab/gitaly/internal_sockets/ruby.1"],"supervisor.name":"gitaly-ruby.1","supervisor.pid":318,"time":"2021-04-28T01:23:10.578Z"}
{"level":"info","msg":"starting RSS monitor","supervisor.name":"gitaly-ruby.0","supervisor.rss_threshold":209715200,"time":"2021-04-28T01:23:10.589Z"}
{"level":"warning","msg":"grpc: addrConn.createTransport failed to connect to {/var/opt/gitlab/gitaly/internal_sockets/ruby.0  \u003cnil\u003e 0 \u003cnil\u003e}. Err: connection error: desc = \"transport: Error while dialing dial unix /var/opt/gitlab/gitaly/internal_sockets/ruby.0: connect: connection refused\". Reconnecting...","pid":281,"system":"system","time":"2021-04-28T01:23:10.606Z"}
{"level":"info","msg":"starting RSS monitor","supervisor.name":"gitaly-ruby.1","supervisor.rss_threshold":209715200,"time":"2021-04-28T01:23:10.615Z"}
{"level":"warning","msg":"grpc: addrConn.createTransport failed to connect to {/var/opt/gitlab/gitaly/internal_sockets/ruby.1  \u003cnil\u003e 0 \u003cnil\u003e}. Err: connection error: desc = \"transport: Error while dialing dial unix /var/opt/gitlab/gitaly/internal_sockets/ruby.1: connect: connection refused\". Reconnecting...","pid":281,"system":"system","time":"2021-04-28T01:23:10.622Z"}
{"address":"/var/opt/gitlab/gitaly/gitaly.socket","level":"info","msg":"listening at unix address","time":"2021-04-28T01:23:10.798Z"}
{"address":"/var/opt/gitlab/gitaly/internal_sockets/internal.sock","level":"info","msg":"listening at unix address","time":"2021-04-28T01:23:10.799Z"}
{"address":"localhost:9236","level":"info","msg":"starting prometheus listener","time":"2021-04-28T01:23:10.834Z"}
{"level":"info","msg":"maintenance: daily scheduled","pid":281,"scheduled":"2021-04-28T12:00:00Z","time":"2021-04-28T01:23:10.839Z"}
{"level":"info","msg":"PID 317 BUNDLE_GEMFILE=/opt/gitlab/embedded/service/gitaly-ruby/Gemfile","supervisor.args":["bundle","exec","bin/ruby-cd","/var/opt/gitlab/gitaly","/opt/gitlab/embedded/service/gitaly-ruby/bin/gitaly-ruby","281","/var/opt/gitlab/gitaly/internal_sockets/ruby.0"],"supervisor.name":"gitaly-ruby.0","time":"2021-04-28T01:23:15.747Z"}
{"level":"info","msg":"PID 318 BUNDLE_GEMFILE=/opt/gitlab/embedded/service/gitaly-ruby/Gemfile","supervisor.args":["bundle","exec","bin/ruby-cd","/var/opt/gitlab/gitaly","/opt/gitlab/embedded/service/gitaly-ruby/bin/gitaly-ruby","281","/var/opt/gitlab/gitaly/internal_sockets/ruby.1"],"supervisor.name":"gitaly-ruby.1","time":"2021-04-28T01:23:15.861Z"}
{"level":"warning","msg":"grpc: addrConn.createTransport failed to connect to {/var/opt/gitlab/gitaly/internal_sockets/ruby.1  \u003cnil\u003e 0 \u003cnil\u003e}. Err: connection error: desc = \"transport: Error while dialing dial unix /var/opt/gitlab/gitaly/internal_sockets/ruby.1: connect: connection refused\". Reconnecting...","pid":281,"system":"system","time":"2021-04-28T01:23:25.653Z"}
{"level":"warning","msg":"grpc: addrConn.createTransport failed to connect to {/var/opt/gitlab/gitaly/internal_sockets/ruby.0  \u003cnil\u003e 0 \u003cnil\u003e}. Err: connection error: desc = \"transport: Error while dialing dial unix /var/opt/gitlab/gitaly/internal_sockets/ruby.0: connect: connection refused\". Reconnecting...","pid":281,"system":"system","time":"2021-04-28T01:23:25.653Z"}
{"level":"warning","msg":"grpc: addrConn.createTransport failed to connect to {/var/opt/gitlab/gitaly/internal_sockets/ruby.1  \u003cnil\u003e 0 \u003cnil\u003e}. Err: connection error: desc = \"transport: Error while dialing dial unix /var/opt/gitlab/gitaly/internal_sockets/ruby.1: connect: connection refused\". Reconnecting...","pid":281,"system":"system","time":"2021-04-28T01:23:40.655Z"}
{"level":"warning","msg":"grpc: addrConn.createTransport failed to connect to {/var/opt/gitlab/gitaly/internal_sockets/ruby.0  \u003cnil\u003e 0 \u003cnil\u003e}. Err: connection error: desc = \"transport: Error while dialing dial unix /var/opt/gitlab/gitaly/internal_sockets/ruby.0: connect: connection refused\". Reconnecting...","pid":281,"system":"system","time":"2021-04-28T01:23:40.655Z"}
{"level":"warning","msg":"grpc: addrConn.createTransport failed to connect to {/var/opt/gitlab/gitaly/internal_sockets/ruby.0  \u003cnil\u003e 0 \u003cnil\u003e}. Err: connection error: desc = \"transport: Error while dialing dial unix /var/opt/gitlab/gitaly/internal_sockets/ruby.0: connect: connection refused\". Reconnecting...","pid":281,"system":"system","time":"2021-04-28T01:23:55.674Z"}
{"level":"warning","msg":"grpc: addrConn.createTransport failed to connect to {/var/opt/gitlab/gitaly/internal_sockets/ruby.1  \u003cnil\u003e 0 \u003cnil\u003e}. Err: connection error: desc = \"transport: Error while dialing dial unix /var/opt/gitlab/gitaly/internal_sockets/ruby.1: connect: connection refused\". Reconnecting...","pid":281,"system":"system","time":"2021-04-28T01:23:55.675Z"}
{"level":"warning","msg":"grpc: addrConn.createTransport failed to connect to {/var/opt/gitlab/gitaly/internal_sockets/ruby.1  \u003cnil\u003e 0 \u003cnil\u003e}. Err: connection error: desc = \"transport: Error while dialing dial unix /var/opt/gitlab/gitaly/internal_sockets/ruby.1: connect: connection refused\". Reconnecting...","pid":281,"system":"system","time":"2021-04-28T01:24:10.681Z"}
{"level":"warning","msg":"grpc: addrConn.createTransport failed to connect to {/var/opt/gitlab/gitaly/internal_sockets/ruby.0  \u003cnil\u003e 0 \u003cnil\u003e}. Err: connection error: desc = \"transport: Error while dialing dial unix /var/opt/gitlab/gitaly/internal_sockets/ruby.0: connect: connection refused\". Reconnecting...","pid":281,"system":"system","time":"2021-04-28T01:24:10.681Z"}
{"correlation_id":"24e5959d3e0e3713b24dcdb4fef76397","grpc.code":"OK","grpc.meta.auth_version":"v2","grpc.meta.client_name":"gitlab-sidekiq","grpc.meta.deadline_type":"unknown","grpc.method":"Check","grpc.request.deadline":"2021-04-28T02:20:48.008Z","grpc.request.fullMethod":"/grpc.health.v1.Health/Check","grpc.service":"grpc.health.v1.Health","grpc.start_time":"2021-04-28T02:20:38.608Z","grpc.time_ms":0.045,"level":"info","msg":"finished unary call with code OK","peer.address":"@","pid":281,"span.kind":"server","system":"grpc","time":"2021-04-28T02:20:38.608Z"}
{"correlation_id":"24e5959d3e0e3713b24dcdb4fef76397","grpc.code":"OK","grpc.meta.auth_version":"v2","grpc.meta.client_name":"gitlab-sidekiq","grpc.meta.deadline_type":"unknown","grpc.method":"RepositoryExists","grpc.request.deadline":"2021-04-28T02:21:00.019Z","grpc.request.fullMethod":"/gitaly.RepositoryService/RepositoryExists","grpc.request.glProjectPath":"gitlab-instance-518f5325/Monitoring.wiki","grpc.request.glRepository":"wiki-1","grpc.request.repoPath":"@hashed/6b/86/6b86b273ff34fce19d6b804eff5a3f5747ada4eaa22f1d49c01e52ddb7875b4b.wiki.git","grpc.request.repoStorage":"default","grpc.request.topLevelGroup":"@hashed","grpc.service":"gitaly.RepositoryService","grpc.start_time":"2021-04-28T02:20:50.728Z","grpc.time_ms":0.226,"level":"info","msg":"finished unary call with code OK","peer.address":"@","pid":281,"span.kind":"server","system":"grpc","time":"2021-04-28T02:20:50.729Z"}
{"level":"info","msg":"finished tempdir cleaner walk","storage":"default","time":"2021-04-28T02:23:10.545Z","time_ms":0}
